cli/src/aws_sso_util/populate_profiles.py

```diff
@@ -149,6 +149,23 @@ def trim_formatter(i, n, **kwargs):
         return formatter(i, n, **kwargs)
     return trim_formatter
 
+def get_field_name_case_formatter(field, transform, formatter):
+    def case_formatter(i, n, **kwargs):
+        LOGGER.debug("evaluating case transform(%s)", transform)
+        LOGGER.debug("* in  = %s", kwargs[field])
+        if transform == 'capitalize':
```

Comment:
With the change to this function to operate directly on the fields, I think this looks more like:
```python
for field, transform in [("account_name", account_name_case), ("role_name", role_name_case)]:
    if not transform:
        continue
    if transform == "capitalize":
        kwargs[field] = kwargs[field].capitalize()
    # etc.
```
Make it more direct rather than the generic transform application happening here.

Reply:
heh, i just did something just like that. let me know if it works for you.

Reply:
oh, do you want me to get rid of the lambda?

Reply:
yeah, I don't think the lambda is necessary. Also, I think you're missing a `.items()` for iterating over that map

Reply:
whoops! thought i'd tested the last update, but apparently not. updated again, removed the lambdas, fixed that bug.

Reply:
Let's also add an `else` that raises `ValueError` for an unknown transform type. The argument processing should prevent it, but I don't like to have unhandled cases